core/src/scene/sceneLoader.cpp

```diff
@@ -1278,11 +1335,14 @@ bool SceneLoader::parseStyleUniforms(const Node& value, Scene& scene, StyleUnifo
         } else {
             const std::string& strVal = value.Scalar();
             styleUniform.type = "sampler2D";
-            std::shared_ptr<Texture> texture = scene.getTexture(strVal);
 
-            if (!texture && !loadTexture(strVal, scene)) {
-                LOGW("Can't load texture with name %s", strVal.c_str());
-                return false;
+            if (scene) {
```

Comment:
https://github.com/tangrams/tangram-es/blob/74a467a932b8e6dc0af6960ae667e8fb5f795922/core/src/scene/importer.cpp#L217

Need to reuse it to determine type of uniforms to normalize texture paths!